server/utils/smtp.ts

```diff
@@ -29,13 +30,16 @@ export async function sendEmail({
     tls: config.get('smtp.tls'),
   })
 
-  const info = await transporter.sendMail({
-    from: config.get('smtp.from'), // sender address
-    to,
-    subject,
-    text,
-    html,
-  })
-
-  logger.info({ messageId: info.messageId }, 'Email sent')
+  try {
+    const info = await transporter.sendMail({
+      from: config.get('smtp.from'), // sender address
+      to,
+      subject,
+      text,
+      html,
+    })
+    logger.info({ messageId: info.messageId }, 'Email sent')
+  } catch (err) {
+    throw GenericError({}, 'Error sending email to reviewers', 500)
```

Comment:
I think you should _also_ run `logger.error()` here with the original `err` to avoid losing that information.  Bunyan has custom loggers to help you log errors:

```

log.info(err);  // Special case to log an `Error` instance to the record.
                // This adds an "err" field with exception details
                // (including the stack) and sets "msg" to the exception
                // message.
log.info(err, 'more on this: %s', more);
                // ... or you can specify the "msg".
```

Reply:
Since this is likely to be a non-fatal error, you should use `logger.warn()` instead of `logger.error()`.